core/src/main/resources/defaults/blueprints/cdp-sdx-721.bp

```diff
@@ -222,6 +222,10 @@
           }
         ],
         "serviceConfigs": [
+          {
+            "name": "hbase_wal_dir",
+            "value": "/hbase-wals"
```

Comment:
Does this directory need to exist in HDFS? Is it automatically created? If not, should we add code to make sure that this directory exists?

Reply:
Blueprint just needs to add that configuration. CM will create this directory automatically. I have verified this at the host. With this change only, the folder "/hbase-wals" was created, and it contains data.

[root@lina-0804-aws-s3-1-master1 bin]# hdfs dfs -ls /
Found 4 items
drwxr-xr-x   - hbase hbase               0 2020-08-04 11:41 /hbase-wals
drwxrwxr-x   - solr  solr                0 2020-08-04 11:40 /solr-infra
drwxrwxrwt   - hdfs  supergroup          0 2020-08-04 11:40 /tmp

[root@lina-0804-aws-s3-1-master1 bin]# hadoop fs -du -s -h /hbase-wals
584.6 K  512.6 M  /hbase-wals
drwxr-xr-x   - hdfs  supergroup          0 2020-08-04 15:43 /user

Reply:
Ok sounds good.